licenses.d2iq.yaml

```diff
@@ -81,6 +81,11 @@ resources:
       - url: https://github.com/grafana/loki
         ref: v${image_tag}
         license_path: LICENSE
+  - container_image: docker.io/grafana/loki:2.8.0
```

Comment:
might be easier to cherry-pick https://github.com/mesosphere/kommander-applications/pull/1150 into this PR and remove the old loki image in the same PR